shapeshift/lib
# feat: detect completed txs and throw error on fail

Detect completed & failed transactions.

Will have a followup pr to translate error toast on trade error on web

To test:

link to web. enable thor. trade into a coin that completed quickly (eth to ltc is good) and see the trade spinner until it completes
